### DISCUSSION TITLE
Keystrokes to open Github repository

### DISCUSSION THREAD

**Discussion by pietvo** [magit/magit#4590] [Ideas]
I seem to remember that I had some keystrokes in Magit to browse the github repository of 'origin', something like 'H H'.
However, I can't find that anymore, and moreover, the 'H' key is now about section info.

Does anyone know what kind of command this may have been, or which function it could be so that I can rebind it to another key sequence?

**Comment by tarsius** [comment]
`C-c C-w` (w for web) on the merge (or rebase) line in the header.

**Reply by pietvo** [reply]
> `C-c C-w` (w for web) on the merge (or rebase) line in the header.

Thanks. I didn't know this. It is not the command that I had in mind, because that could be given anywhere in the Magit status buffer. But your answer took me in the right direction, and I found `forge-browse-remote` as the command to use. 
I have now

```
(with-eval-after-load 'magit-remote
  (transient-append-suffix 'magit-remote "k"
  '("H" "Browse Github" forge-browse-remote)))
```
and now M H brings me to the Github repository. I still don't know where my original command came from, however.

**Reply by tarsius** [reply]
I've added a note to my todo list to make more of these commands readily available and/or improve dwiming.
